### ISSUE TITLE
Add long help for each command

### ISSUE THREAD

**Issue by twpayne** [twpayne/chezmoi#6]
Currently commands a loosely documented in `README.md`, or not at all. Each command should have  long documentation as well.

**Comment by kkirsche** [comment]
Is the idea here that each command would replicate or have similar documentation to the README.md? For example, on the `addCommand`, would the ideal code after look similar to:

```
var addCommand = &cobra.Command{
        Use:   "add",
        Args:  cobra.MinimumNArgs(1),
        Short: "Add an existing file, directory, or symlink to the source state",
        Long: `To manage an existing file with chezmoi:

$ chezmoi add ~/.bashrc

This will create the source directory ~/.local/share/chezmoi with permissions 0600 where chezmoi will store the source file
state. If the file you will be storing will leverage chezmoi's templating features, you can add the file as a template
using:

$ chezmoi add -T ~/.bashrc

Chezmoi will take this opportunity to replace any instances of known variables (as configured in your config file) with
the variable syntax.

To ensure that a file exists, even if empty, you can use the -e flag like so:

$ chezmoi add -e ~/.config/example/.keep

For more information, visit https://github.com/twpayne/chezmoi
`,
        RunE: makeRunE(config.runAddCommand),
}
```

?

**Comment by kkirsche** [comment]
Hm, this is one then that I think will be harder and potentially not ideal for a first issue then. Specifically, because of the goal of covering corner cases, which I don't expect a new contributor to know. For me, that would be the biggest barrier to completing this task as a new user of the tool.

**Comment by twpayne** [comment]
I'm using the "Good first issue" label for new _contributors_, not new _users_. They're generally issues that don't require too much coding.

**Comment by twpayne** [comment]
Fixed in #258.
